README.md

```diff
@@ -5,54 +5,37 @@
 [![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/psf/black)
 [![License](https://img.shields.io/github/license/dcs4cop/xcube-smos)](https://github.com/xcube-dev/xcube-stac/blob/main/LICENSE)
 
-`xcube-stac` is a Python package and
-[xcube plugin](https://xcube.readthedocs.io/en/latest/plugins.html) that adds a
-[data store](https://xcube.readthedocs.io/en/latest/api.html#data-store-framework)
-named `stac` to xcube. The data store is used to access data from the
-[STAC - SpatioTemporal Asset Catalogs](https://stacspec.org/en/).
+`xcube-stac` is a Python package and a [xcube plugin](https://xcube.readthedocs.io/en/latest/plugins.html)
+that provides a [data store](https://xcube.readthedocs.io/en/latest/api.html#data-store-framework)
+for accessing data from [STAC (SpatioTemporal Asset Catalogs)](https://stacspec.org/en/).
+
 
 ## Table of contents
-1. [Setup](#setup)
-   1. [Installing the xcube-stac plugin from the repository](#install_source)
-2. [Overview](#overview)
+1. [Overview](#overview)
    1. [General structure of a STAC catalog](#stac_catalog)
    2. [General functionality of xcube-stac](#func_xcube_stac)
+2. [Setup](#setup)
+   1. [Installing the xcube-stac plugin](#install_plugin)
+   2. [Getting S3 credentials for CDSE data access ](#cdse_s3)
 3. [Introduction to xcube-stac](#intro_xcube_stac)
    1. [Overview of Jupyter notebooks](#overview_notebooks)
    2. [Getting started](#getting_started)
 4. [Testing](#testing)
    1. [Some notes on the strategy of unit-testing](#unittest_strategy)
 
-## Setup <a name="setup"></a>
-
-### Installing the xcube-stac plugin from the repository <a name="install_source"></a>
-
-Installing xcube-stac directly from the git repository, clone the repository,
-direct into `xcube-stac`, and follow the steps below:
-
-```bash
-conda env create -f environment.yml
-conda activate xcube-stac
-pip install .
-```
-
-This installs all the dependencies of `xcube-stac` into a fresh conda
-environment, then installs xcube-stac into this environment from the
-repository.
-
 ## Overview <a name="overview"></a>
```

Comment:
You should need the HTML anchors, afaik. At least I never did and linking still works. Anchor names are generated from head titles by making them lowercase and replacing spaces by hyphens.